package.json

```diff
@@ -17,7 +17,7 @@
     }
   ],
   "scripts": {
-    "test": "ava && xo",
+    "test": "LD_LIBRARY_PATH=./libui/build/out ava && xo",
```

Comment:
It's not needed on OS X. The earlier libui.js has this in its `binding.gyp`:

`"libraries": ["-Wl,-rpath, ../libui/out/libui.so.0"]`

Apparently that might make it work without `LD_LIBRARY_PATH` on some Linux distributions, but not on my Ubuntu. The equivalent for OS X in this PR does work. I spent some time trying to understand a linker option `-Wl,--enable-new-dtags` that might have fixed the issue on Linux but ultimately it didn't help.